graphql_compiler/macros/macro_edge/directives.py

```diff
@@ -36,9 +38,23 @@
     ],
 )
 
-
+# Directives reserved for macro edges
 MACRO_EDGE_DIRECTIVES = (
     MacroEdgeDirective,
     MacroEdgeDefinitionDirective,
     MacroEdgeTargetDirective,
 )
+
+# Directives required present in a macro edge definition
+DIRECTIVES_REQUIRED_IN_MACRO_EDGE_DEFINITION = frozenset({
+    MacroEdgeDefinitionDirective,
+    MacroEdgeTargetDirective
+})
+
+# Directives tolerated without restrictions within a macro edge definition
+DIRECTIVES_ALLOWED_IN_MACRO_EDGE_DEFINITION = frozenset({
```

Comment:
This code is inconsistent. The set above uses actual directives while in this set you include strings yet both names start with "DIRECTIVE". Please either change the naming to DIRECTIVE_NAMES_ALLOWED_IN_MACRO_EDGE_DEFINTION or change the types of the set elements.

Reply:
Good point. This is fixed in the next commit.